src/allotment.tsx

```diff
@@ -454,6 +457,12 @@ const Allotment = forwardRef<AllotmentHandle, AllotmentProps>(
       }
     }, []);
 
+    useEffect(() => {
+      if (gap) {
+        document.documentElement.style.setProperty("--gap-size", gap + "px");
+      }
+    }, [gap]);
```

Comment:
Agreed on both points. I've not progressed this PR because it doesn't play nice with the 'real' dimensions of the panes. If you ask for a minSize of 10 then it won't account for the gap size. Which is a long-winded way of saying I need to go in and change the guts of how allotment works. Which I want to do but don't have the time right now.

The `gap === 0` point is a really good one though. Just the sort of thing a code review would catch 😁